exasol/ds/sandbox/runtime/ansible/roles/entrypoint/files/entrypoint.py

```diff
@@ -153,27 +175,58 @@ def disable_core_dumps():
     resource.setrlimit(resource.RLIMIT_CORE, (0, 0))
 
 
-def sleep_inifinity():
+def sleep_infinity():
     while True:
         time.sleep(1)
 
 
+class User:
+    def __init__(self, name: str):
+        self.name = name
+        self._id = None
+
+    @property
+    def id(self):
+        if self._id is None:
+            self._id = pwd.getpwnam(self.name).pw_uid
+        return self._id
+
+    def own(self, path: str):
+        if Path(path).exists():
+            unchanged_gid = -1
+            os.chown(path, self.id, unchanged_gid)
```

Comment:
owner of the docker socket, should still be root, but we want that the group of the socket is the docker group, which the jupyter user is part of